README.md

```
@@ -1,5 +1,66 @@
+
+Build Docker image
+
+```shell
+mvn install -DskipTests -Dpowsybl.docker.install
+```
+
+---
+
+## Interactions with Other Microservices
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)